ros2_foxglove_bridge/src/ros2_foxglove_bridge.cpp

```diff
@@ -846,7 +856,9 @@ void FoxgloveBridge::fetchAsset(const std::string& uri, uint32_t requestId,
     response.errorMessage = "Failed to retrieve asset " + uri;
   }
 
-  _server->sendFetchAssetResponse(clientHandle, response);
+  if (_server) {
+    _server->sendFetchAssetResponse(clientHandle, response);
```

Comment:
Is this method thread safe?

Reply:
Yes. It's basically the same as `_server->sendMessage` which is also called by multiple threads (if subscribed to multiple topics)